docs/superpowers/plans/2026-06-07-vote-system-rebuild.md

```
@@ -0,0 +1,780 @@
+
+    return NextResponse.json(Object.values(grouped) as VoteResult[]);
+  } catch {
+    return NextResponse.json([] as VoteResult[]);
+  }
+};
+```
+
+- [ ] **Step 2: 동작 확인**
+
+dev 서버 실행 중 브라우저에서:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)